src/renderer/entities/network/model/network-model.ts

```diff
@@ -221,26 +213,26 @@ sample({
   source: {
     chains: $chains,
     connections: $connections,
+    metadata: $metadata,
```

Comment:
why do you need `metadata` here and `$metadata` at all? You use `networkUtils.getNewestMetadata` to get metadata, not `$metadata`

Reply:
`networkUtils.getNewestMetadata` works with existing metadata
At first we load all metadata from db
Then pass the newest version to `createCachedProvider`
Previously we requested metadata from db during `send` method inside `createCachedProvider` which is bad